libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBTableColumn.java

```diff
@@ -122,7 +122,7 @@ public class DBTableColumn implements DBObject, DBObjectWarningDescriptor {
     private String collationName;
 
     /**
-     * MySQL special
+     * Default value for the column
```

Comment:
From the name, `genExpression` refers to the generation column's expression, not the default value. Oracle uses `COL_DATA_DEFAULT` to represent `genExpression`, which is just an implementation detail in Oracle's data dictionary views, but it does not mean that this field represents a default value.

Reply:
OK，I have modified the comment.